pkgs/tools/typesetting/tex/mftrace/default.nix

```diff
@@ -45,8 +45,8 @@ let self = stdenv.mkDerivation rec {
   # experimental texlive.combine support
   # (note that only the bin/ folder will be combined into texlive)
   passthru.tlType = "bin";
-  passthru.pkgs = [ self ] ++
-    (with texlive; kpathsea.pkgs ++ t1utils.pkgs ++ metafont.pkgs);
+  passthru.tlDeps = with texlive; [ kpathsea t1utils metafont ];
+  passthru.pkgs = [ self ];
```

Comment:
```suggestion
  passthru = {
    tlType = "bin";
    tlDeps = with texlive; [ kpathsea t1utils metafont ];
    pkgs = [ self ];
  };
```

Reply:
Forgot to do this in my first forced push, I have updated this as well in all four packages.